src/app/entry/extendedDescriptorLanguage.ts

```diff
@@ -86,6 +91,7 @@ const superService: ExtendedDescriptorLanguage = {
 
 const superGalaxy: ExtendedDescriptorLanguage = {
   value: 'gxformat2',
+  shortFriendlyName: 'GalaxyV2',
```

Comment:
Unfortunately, the shortFriendlyName might have to be "Galaxy"
The friendlyName "Galaxy Workflow Format"

For example, the path *.ga would be format 1, *.yml or *.yaml would be format 2. 
But we don't really track this and that not tracking it in this bean would seem to be consistent with WDL which only gets one enum value regardless of whether it is WDL draft-2 or WDL v1